pallets/manta-pay/src/lib.rs

```diff
@@ -201,6 +201,9 @@ pub mod pallet {
         #[transactional]
         pub fn to_private(origin: OriginFor<T>, post: TransferPost) -> DispatchResultWithPostInfo {
             let origin = ensure_signed(origin)?;
+            for source in post.sources.iter() {
+                ensure!(asset_value_decode(*source) > 0u128, Error::<T>::BalanceLow);
+            }
```

Comment:
the front-end doesn't do 0 value txs anymore, right ?

Reply:
The frontend UI has 0 value checking, but sdk don't have this check now.

Reply:
Yeah, IMHO, it's best to add this simple check on every level of the stack